app-text/zathura/zathura-0.5.4.ebuild

```diff
@@ -13,38 +13,50 @@ if [[ ${PV} == *9999 ]]; then
 	EGIT_REPO_URI="https://git.pwmt.org/pwmt/${PN}.git"
 	EGIT_BRANCH="develop"
 else
-	SRC_URI="https://github.com/pwmt/zathura/archive/${PV}.tar.gz -> ${P}.tar.gz"
-	KEYWORDS="amd64 arm ~riscv x86 ~amd64-linux ~x86-linux"
+	SRC_URI="
+		https://github.com/pwmt/zathura/archive/${PV}.tar.gz -> ${P}.tar.gz
+		https://cdn.turret.cyou/~turret/distfiles/${PN}/${P}-manpages.tar.xz
+	"
+	KEYWORDS="~amd64 ~arm ~riscv ~x86 ~amd64-linux ~x86-linux"
 fi
 
 LICENSE="ZLIB"
-SLOT="0"
-IUSE="doc +magic seccomp sqlite synctex test"
+SLOT="0/5.6"
```

Comment:
In a followup, consider adding the details from the commit message as a comment above `SLOT` too.